packages/webpack/lib/utils/compiler.js

```diff
@@ -77,9 +77,10 @@ exports.createWatchCompiler = createWatchCompiler;
 process.on('message', (message) => {
   if (message.name !== 'start') return;
   const { buildConfigArgs, overrides, options } = message;
-  const { handleArguments, getBuildConfig } = initialize(overrides);
-  handleArguments(options);
-  const webpackConfig = getBuildConfig(...buildConfigArgs);
+  const webpackConfig = configure(overrides, {
+    ...options,
+    _: [],
```

Comment:
Is this not obsolete as we have the [default](https://github.com/untool/untool/pull/355/files#diff-a4275fbdb28414118289ba0f39b70d04R43)?

Reply:
Nope, we explicitly want to set it to an empty array to prevent the log mixin to log "running '...' in '...' mode". Because the options that we get passed in here, do contain the command of the parent process.